docs/pages/introduction/index.js

```diff
@@ -33,62 +33,55 @@ export class Introduction extends LitElement {
 
   render() {
     return wrap({
-      title: "Introduction",
+      title: 'Introduction',
       intro:
-        "We offer a library of web components that is build upon the design system of Digitaal Vlaanderen. It is designed to work with any framework, or even without one. We guarantee consistency between the components.",
+        'We offer a library of web components that is build upon the design system of Digitaal Vlaanderen. It is designed to work with any framework, or even without one. We guarantee consistency between the components.',
       children: html`<section is="vl-region">
         <div is="vl-layout">
           <div is="vl-grid" data-vl-is-stacked>
             ${block({
-              title: "Getting started",
+              title: 'Getting started',
               children: html`<p>
                 To get started, you need to install our package. To do so, run
-                <vl-typography
-                  ><code>npm install uig-webcomponents</code></vl-typography
+                <vl-typography><code>npm install uig-webcomponents</code></vl-typography
                 >. Once the installation is done, you are ready to go.
               </p>`,
             })}
             ${block({
-              title: "Usage",
+              title: 'Usage',
               children: html`
                 <p>
-                  You can now import and use our components in your project.
-                  Each component has a docs page, where you find what needs to
-                  be imported in order to use it.
+                  You can now import and use our components in your project. Each component has a docs page, where you
+                  find what needs to be imported in order to use it.
                 </p>
               `,
             })}
             ${block({
-              title: "Migrating from legacy packages",
+              title: 'Migrating from legacy packages',
               children: html`
                 <p>
-                  If you are migrating from the legacy packages to this package,
-                  we recommend to carefully read the docs and the changelog.
-                  Every breaking change across from the legacy package is
-                  mentioned in the changelog with a '&#x1F4A5;'. We use semantic
-                  versioning for our own changes.
+                  If you are migrating from the legacy packages to this package, we recommend to carefully read the docs
+                  and the changelog. Every breaking change across from the legacy package is mentioned in the changelog
+                  with a '&#x1F4A5;'. We use semantic versioning for our own changes.
+                </p>
+                <br />
```

Comment:
Die `br` gaat in principe niet de juiste margin zetten van het design system. Om dat te bekomen gebruik je de `vl-typography`. Het voorbeeldje geeft de juiste margin. I know, raar opgezet in het verleden. Ik begrijp niet waarom die margin niet gewoon in de body styles zit die je importeert in elk project.

```
<vl-typography>
  <p />
  <p />
</vl-typography>